python/ml4ir/applications/classification/tests/test_classification_model.py

```diff
@@ -42,7 +42,7 @@ def test_group_metrics_df(self):
         Test the dimensions of the grouped metrics
         """
         # Metrics cardinality and names
-        metrics = self.classification_pipeline.metrics_keys  # Metrics during training
+        metrics = ["loss"] + self.classification_pipeline.metrics_keys  # Metrics during training
```

Comment:
I'm not certain where this is defined,but maybe self.classification_pipeline.metrics_keys.insert(0, "loss")